cpu/efm32/Makefile.include

```diff
@@ -14,6 +14,12 @@ ifeq ($(CPU_ARCH),cortex-m0plus)
   USEPKG += cmsis-dsp
 endif
 
+# Include CPU family module
```

Comment:
Tiny-tiny-tiny comment: most comments in RIOT-OS don't start with a capital letter.

Reply:
Never noticed this :) I added a fixup.